geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -138,9 +138,9 @@ <h3>
         {% if document.has_transcription and user.is_authenticated %}
             <section class="transcription-link">
                 {% for ed in document.digital_editions %}
-                    {% if ed.content.text %}
+                    {% comment %} {% if ed.content.text %}
                         <p><a href="{% url "corpus:document-transcription-text" document.pk ed.pk %}">Download {% for auth in ed.source.authorship_set.all %}{% include "snippets/comma.html" %}{{ auth.creator.last_name }}{% empty %}[unknown]{% endfor %}'s edition.</a></p>
-                    {% endif %}
+                    {% endif %} {% endcomment %}
                 {% endfor %}
```

Comment:
commented out because it doesn't work anymore?

Reply:
Right, there's no `content.text` now, so needs to be updated in the upcoming #999 work, but commented out temporarily while this is a WIP.

Reply:
this was a provisional solution, let's keep it commented out and mark as something to address later (we may remove it entirely in favor of the cut + paste we still need to implement)